test/augmentation/test_container.py

```diff
@@ -302,6 +302,8 @@ def test_random_flips(self, device, dtype):
         assert_close(out_ver[1], expected_bbox_vertical_flip)
         assert_close(out_hor[1], expected_bbox_horizontal_flip)
 
+    # TODO: fix the bug in #1520 and remove skip
+    @pytest.mark.skip(reason="Padding is not well computed")
```

Comment:
@twsl @shijianjian added this pending of https://github.com/kornia/kornia/issues/1520